### ISSUE TITLE
app's jacoco-it.exec seems invalid to report-aggregate

### ISSUE THREAD

**Issue by SamPanza** [SamPanza/EmKa#5]
```
[INFO] --- jacoco-maven-plugin:0.8.10:prepare-agent-integration (default) @ emka-app ---
[INFO] argLine set to -javaagent:~/.m2/repository/org/jacoco/org.jacoco.agent/0.8.10/org.jacoco.agent-0.8.10-runtime.jar=destfile=~/w/EmKa/app/target/jacoco-it.exec,dumponexit=true
```

```
[INFO] --- jacoco-maven-plugin:0.8.10:report-aggregate (default) @ emka-site ---
[INFO] Loading execution data file ~/w/EmKa/core/target/jacoco.exec
[INFO] Loading execution data file ~/w/EmKa/junit/target/jacoco.exec
[INFO] Loading execution data file ~/w/EmKa/app/target/jacoco-it.exec
```

```
[ERROR] Failed to execute goal org.jacoco:jacoco-maven-plugin:0.8.10:report-aggregate (default) on project emka-site: An error has occurred in JaCoCo Aggregate report generation.: Error while creating report: null: EOFException -> [Help 1]
```

**Comment by SamPanza** [comment]
Actually, the file is OK -- jacococli generates report successfully

**Comment by SamPanza** [comment]
Happens within IJ, regular `mvn verify` works

**Comment by SamPanza** [comment]
> Happens within IJ, regular `mvn verify` works

GH build fails too

**Comment by SamPanza** [comment]
Fixed, see this commit 1e7d98d
